src/tally.cpp

```diff
@@ -423,37 +440,59 @@ std::string pyne::Tally::mcnp(int tally_index, std::string mcnp_version) {
 
+  // TODO this could be cleaner
+  std::string mat_num = "?";
+  if ( material ) {
+    int mat_id = material->metadata["mat_number"].asInt();
+    // TODO - need to use decrepid C++98 method
+    mat_num = std::to_string(mat_id);
+  }
+
   // neednt check entity type
   if (entity_type.find("Surface") != std::string::npos) {
     if (tally_type.find("Current") != std::string::npos) {
       output << "F"<< tally_index <<"1:" << particle_token 
-	     << " " << entity_id << std::endl;
+       << " " << entity_id << std::endl;
       if (entity_size > 0.0)
-	output << "SD"<<tally_index <<"1 " << entity_size << std::endl;
+  output << "SD"<<tally_index <<"1 " << entity_size << std::endl;
       // normalisation
       if (normalization > 1.0)
-	output << "FM" << tally_index << "1 " << normalization << std::endl;
+  output << "FM" << tally_index << "1 " << normalization << std::endl;
     } else if (tally_type.find("Flux") != std::string::npos) {
       output << "F"<< tally_index <<"2:" << particle_token 
-	     << " " << entity_id << std::endl;
+       << " " << entity_id << std::endl;
       if (entity_size > 0.0)
-	output << "SD"<<tally_index <<"2 " << entity_size << std::endl;
+  output << "SD"<<tally_index <<"2 " << entity_size << std::endl;
       // normalisation
       if(normalization > 1.0)
-	output << "FM" << tally_index << "2 " << normalization << std::endl;
+  output << "FM" << tally_index << "2 " << normalization << std::endl;
 
     }
   } else if (entity_type.find("Volume") != std::string::npos) {
     if (tally_type.find("Flux") != std::string::npos) {
       output << "F"<< tally_index <<"4:" << particle_token << " " 
-	     << entity_id << std::endl;
+       << entity_id << std::endl;
       if (entity_size > 0.0)
-	output << "SD"<<tally_index <<"4 " << entity_size << std::endl;
+  output << "SD"<<tally_index <<"4 " << entity_size << std::endl;
             // normalisation
       if(normalization > 1.0)
-	output << "FM" << tally_index << "4 " << normalization << std::endl;
+  output << "FM" << tally_index << "4 " << normalization << std::endl;
     } else if (tally_type.find("Current") != std::string::npos) {
-      // makes no sense in mcnp
+      // volume current tally makes no sense in mcnp
+
+    } else if (tally_type.find("TBR") != std::string::npos && particle_token == "n" ) {
+        output << "F" << tally_index << "4:" << particle_token << " " << entity_id << std::endl;
+        if(mat_num == "?" ) { std::cout << "mat_number unset for a need fm card" << std::endl;}
+          output << "FM" << tally_index << "4 " << " 1 " << mat_num << " 205 " << std::endl;
+
+    } else if (tally_type.find("DPA") != std::string::npos ) {
+        output << "F" << tally_index << "4:" << particle_token << " " << entity_id << std::endl;
+        if(mat_num == "?" ) { std::cout << "mat_number unset for a need fm card" << std::endl;}
+          output << "FM" << tally_index << "4 " << " 1 " << mat_num << " 444 " << std::endl;
```

Comment:
maybe there is a way to combine those sections, as the only difference between them is a `444` swapped for a `205`